setup.cfg

```diff
@@ -41,7 +41,11 @@ per-file-ignores =
     linkcheck/parser/__init__.py: E402
     tests/__init__.py: F401
 # E501: line too long
+    linkcheck/better_exchook2.py: E501
+    linkcheck/ftpparse.py: E501
     tests/test_ftpparse.py: E501
+# F821 undefined name
+    linkcheck/logger/__init__.py: F821
```

Comment:
Maybe link to the flake8 bug to explain why?  And then we can remove this once the flake8 bug is fixed.
https://github.com/PyCQA/pyflakes/issues/548

Reply:
Done.